tests/attention/test_trtllm_gen_attention.py

```
@@ -1837,3 +1837,534 @@ def test_trtllm_batch_decode_spec(
+        trtllm_compute_precomputed_metadata,
+        trtllm_get_kernel_config,
+    )
+
+    compute_capability = get_compute_capability(torch.device(device="cuda"))
+    if compute_capability[0] != 10:
+        pytest.skip("trtllm-gen precomputed requires SM100/SM103 GPUs.")
+
+    num_qo_heads = num_kv_heads * head_grp_size
+    torch.manual_seed(42)
+
+    q_lens, in_kv_lens, seq_lens = generate_seq_lens_decode(
```

Ruff (0.15.9)

[warning] Unpacked variable `in_kv_lens` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)